content/en/platform/corda/4.7/enterprise/corda-network/the-corda-network.md

```diff
@@ -4,18 +4,18 @@ menu:
   corda-enterprise-4-7:
     identifier: corda-enterprise-4-7-corda-network
     name: "Corda Network Foundation"
-title: Corda Network Foundation
+title: Corda Network
 weight: 500
 ---
 
 
-# Corda Network Foundation and Corda Network
+# Corda Network
 
 
 [Corda Network](https://corda.network/) is a publicly-available internet of Corda nodes operated by network participants. Each
 node is identified by a certificate issued by the network’s identity service, and is also discoverable on a network map.
 
-Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. The Foundation's Board of Directors is made up of the network's initial participants in Spring 2019 and will oversee the Foundation until democratic elections are held. See the [governance model](https://corda.network/governance/governance-guidelines.html) for more detail.
+Corda Network went live in December 2018, and is currently governed by the independent, not-for-profit Corda Network Foundation. See the [governance model](https://corda.network/corda-network-foundation/articles-of-association/) for more detail.
```

Comment:
This hasn't been updated to  https://corda.network/corda-network-foundation/about-the-foundation